tests/test_coordinates.py

```diff
@@ -196,17 +196,16 @@ def test_len(self):
 
     def test_SPH_output_noconvert(self):
         """SPH is a synonym of GEO in spherical"""
-        self.cvals.ticks = Ticktock(['2002-02-02T12:00:00', '2002-02-02T12:00:00'], 'ISO')
         expected = spc.Coords([[2, 45, 90], [3.0, 31, 31]], 'GEO', 'sph', use_irbem=False)
+        expected.ticks = Ticktock(['2002-02-02T12:00:00', '2002-02-02T12:00:00'], 'ISO')
```

Comment:
Do we need this now? Previous iteration didn't even have a ticks on here...shouldn't be needed for GEO->SPH, correct?